pkg/events/recorder.go

```diff
@@ -51,7 +51,7 @@ type recorder struct {
 	cache *cache.Cache
 }
 
-const defaultDedupeTimeout = 2 * time.Minute
+const defaultDedupeTimeout = 5 * time.Minute
```

Comment:
You're changing the default for everything here. I think you just want to change it for the event that you care about